BuildGDK.bat

```diff
@@ -60,39 +59,32 @@ call :MarkStartOfBlock "Create folders"
     md "%CORE_SDK_DIR%\schema"       >nul 2>nul
     md "%CORE_SDK_DIR%\tools"        >nul 2>nul
     md "%CORE_SDK_DIR%\worker_sdk"   >nul 2>nul
-    md "%BUILD_DIR%\code_generation" >nul 2>nul
     md "%BINARIES_DIR%"              >nul 2>nul
 call :MarkEndOfBlock "Create folders"
 
 call :MarkStartOfBlock "Retrieve dependencies"
     spatial package retrieve tools           schema_compiler-x86_64-win32     %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\tools\schema_compiler-x86_64-win32.zip"
     spatial package retrieve schema          standard_library                 %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\schema\standard_library.zip"
-    spatial package retrieve worker_sdk      core-dynamic-x86-win32           %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\worker_sdk\core-dynamic-x86-win32.zip"
-    spatial package retrieve worker_sdk      core-dynamic-x86_64-win32        %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\worker_sdk\core-dynamic-x86_64-win32.zip"
+    spatial package retrieve worker_sdk      c-dynamic-x86-msvc_md-win32      %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86-msvc_md-win32.zip"
+    spatial package retrieve worker_sdk      c-dynamic-x86_64-msvc_md-win32   %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-msvc_md-win32.zip"
     spatial package retrieve worker_sdk      core-dynamic-x86_64-linux        %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\worker_sdk\core-dynamic-x86_64-linux.zip"
-    spatial package retrieve code_generation Improbable.CodeGeneration        %PINNED_CODE_GENERATOR_VERSION% "%BUILD_DIR%\code_generation\Improbable.CodeGeneration.zip"
     rem Download the C++ SDK for its headers, only.
     spatial package retrieve worker_sdk      cpp-static-x86_64-msvc_mtd-win32 %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\cpp-static-x86_64-msvc_mtd-win32.zip"
```

Comment:
You can remove this line.

Reply:
I attempted this yesterday, but the container classes such as improbable::list is included with this package right?

Reply:
Ah yes. How come you guys still use these?

Reply:
In the UnrealObjectRefs we still depend on worker::Option. We might be able to get rid of that dependency soon.

Reply:
I suggest that this gets replaced with `TOptional<...>` in this PR, or you use `TUniquePtr` instead.

Reply:
We 100% are going to making our own Optional or using something else.